tests/commonUtils.sh

```diff
@@ -15,8 +15,8 @@
 #   limitations under the License.
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-RUNTIME_IMAGE='docker.io/istio/runtime:2017-03-22-21.21.48'
-
+MANAGER_HUB_TAG='docker.io/istio,2017-03-22-21.21.48'
```

Comment:
@rshriram: If I understand what @sebastienvas is saying, the Jenkins build will update the `istio.VERSION` file with the appropriate latest images before running the KubeTest.sh. So, if we change the default value that KubeTest.sh is using for the mixer hub and tag to not be hardcoded anymore, but instead read it out of `istio.VERSION`, this will give us a much better default. Do I have that right @sebastienvas?

I don't think this will break stuff worse that it is already. As soon as we switch to @kuat's new image, it's completely broken. I think we need to just commit this and then fix all the incompatibilities as soon as we can.